About.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kamulete training center</title>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+        <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header>
+        <nav class="twitter">
+            <a href="#">Twitter</a>
+            <a href="#">Whatsap</a>
+            <a href="#">training</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="navlogo">
+            <a class="img-logo" href="index.html"><img src="img-folder/logo-image---00.png" alt=""></a>
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+                <li><a class="menu-item" href="#logo">logo</a></li>
+                <li><a class="menu-item" href="#vision">Vision</a></li>
+            </ul>    
+        </nav>
+    </header>    
+        <section class="grtittle">
+            <h1>Kamulete training center(Rwanda)</h1>
+            <hr class="linebar">
+            <div class="centerpar">
+                <p>kamulete trainig center for all people around the world,<br> we invite you to join our training center.
+                    kamulete training center was created in 2023 by kamulete bertin who is the CEO form Rwanda,<br> he found that to train people by knwowing many languages is very helps full and develops creativness in someone.
+                    with that this trainig center give you a diploma after complited the month acording to your program choosen,<br> that diploma is valid in each country around the word.
+                    we have more than 20 teachers by which we have collaboration and they helps us to give some online classes.
+                </p>
+            </div>
+          <div class="contactclass">
+            <p class="contactclass1">please contact us via email for more information.    </p>
+            <p class="contactclass2">email: Kamuletetrainingcenter988@gmail.com<br></p>
+            <p class="contactclass3">@250 kigali , palace (Rwanda)</p>
+          </div>
+        </section>
```

Comment:
[OPTIONAL] Kindly do your best to make this section look like the given design the fonts are different as well as the size 

![about section design](https://user-images.githubusercontent.com/59707859/225983681-a788e6e9-112b-487a-ac5a-eb70722d76c5.PNG)

![about section](https://user-images.githubusercontent.com/59707859/225983743-65cc2945-c4f5-444f-9335-88ddebe2e15c.PNG)